.github/workflows/build.yml

```diff
@@ -100,7 +100,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python3: ${{fromJson(vars.LINUX_PYTHON_VERSIONS || '[6, 7, 8, 9, 10, 11]')}}
+        python3: ${{fromJson(vars.LINUX_PYTHON_VERSIONS || '[7, 8, 9, 10, 11]')}}
```

Comment:
Why is 12 missing here?

Reply:
This steps is setting up the older versions and writing data with them.
We haven't published python 3.12 for the older versions of arcticdb so it can't be added here yet.